static/css/devhub/new-landing/sections/my-addons.less

```diff
@@ -0,0 +1,3 @@
+.DevHub-MyAddons-copy {
+  max-width: 32em;
```

Comment:
I improved the button margins a bit on mobile, the text looks alright from my perspective. Might need some fine-tuning a bit later.